composer.json

```diff
@@ -11,7 +11,7 @@
         "drupal/search_api": "^1.17",
         "drupal/search_api_autocomplete": "^1.3",
         "localgovdrupal/localgov_core": "^2.1",
-        "localgovdrupal/localgov_geo": "^1.0"
+        "localgovdrupal/localgov_geo": "dev-feature/1.x/area-and-ux-autocomplete"
```

Comment:
Hi Ekes, because of this line, I am assuming you are still working on this and don't want us to review it yet.  But please clarify if I am wrong. @ekes

Reply:
They all need to be released at the same time. So they all need reviewing together. The localgov_geo PR is the main one for review (with guidance). There is an ovearching one that installs all the branches for testing together in https://github.com/localgovdrupal/localgov/pull/335